fsharp.spec

```diff
@@ -103,4 +102,5 @@ rm -rf ${RPM_BUILD_ROOT}%{_prefix}/lib/mono/xbuild
 %{_prefix}/lib/mono/gac/policy.3.3.FSharp.Core/
 %{_prefix}/lib/mono/gac/policy.4.0.FSharp.Core/
 %{_prefix}/lib/mono/gac/policy.4.3.FSharp.Core/
+%{_prefix}/lib/mono/xbuild/Microsoft/VisualStudio/
```

Comment:
Oh, I see now that

    %{_prefix}/lib/mono/Microsoft*

is included already.  So what you have added should be sufficient.